src/video.cpp

```diff
@@ -260,13 +260,14 @@ void TextRenderer::Draw(const std::string &text, float x, float y, float max_wid
 /**
  * Estimate the bounding rectangle of a string.
  * @param text Text to estimate.
+ * @param padding Include text padding in the result.
  * @param scale Scaling factor for the text size.
  */
-PointF TextRenderer::EstimateBounds(const std::string &text, float scale) const
+PointF TextRenderer::EstimateBounds(const std::string &text, bool padding, float scale) const
```

Comment:
```suggestion
PointF TextRenderer::EstimateBounds(const std::string &text, bool add_padding, float scale) const
```
or `include_padding` but that's a bit long.

ie rather than saying "space" say "add-space", making it more clear what the flag is doing from, its name.